.github/workflows/integration_test.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
```

Comment:
shouldn't we use the `main` branch for integration testing?

Reply:
Good catch - the branch was renamed but CI wasn't updated.